src/View/Console/DefaultRenderingStrategy.php

```diff
@@ -42,7 +43,7 @@ public function render(MvcEvent $e)
         // marshal arguments
         $response  = $e->getResponse();
 
-        if (empty($result)) {
+        if (empty($result) || !($result instanceof ModelInterface)) {
```

Comment:
Except this comment, all of your other comments relate to the existing test code that I copied. So I'll apply your suggestions to both tests - I was just following what was already there :)